source/installation-guide/wazuh-indexer/step-by-step.rst

```diff
@@ -184,37 +186,44 @@ Cluster initialization involves configuring a default ISM policy, loading new ce
 Testing the cluster installation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Replace ``<WAZUH_INDEXER_IP_ADDRESS>`` and run the following commands to confirm that the installation is successful.
+#. Replace ``<INDEXER_USERNAME>``, ``<INDEXER_PASSWORD>`` and ``<WAZUH_INDEXER_IP_ADDRESS>`` with the Wazuh indexer username and password that you set using the Keystore tool commands:
 
-   .. code-block:: console
+  .. code-block:: console
```

Comment:
```suggestion
    .. code-block:: console
```

Reply:
Done in [Fix new numbering problems](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/641e5635dfba6d4fa5bb6ef08cba14cfbd40fc82) commit